core/iwasm/interpreter/wasm.h

```diff
@@ -536,10 +536,7 @@ typedef struct WASMTableImport {
 typedef struct WASMMemoryImport {
     char *module_name;
     char *field_name;
-    uint32 flags;
-    uint32 num_bytes_per_page;
-    uint32 init_page_count;
-    uint32 max_page_count;
+    WASMMemory memory;
```

Comment:
How about here changing it to `WASMMemoryType mem_type`, and changing WASMMemory definition in the above to:
```C
typedef struct WASMMemory {
    uint32 flags;
    uint32 num_bytes_per_page;
    uint32 init_page_count;
    uint32 max_page_count;
} WASMMemory, WASMMemoryType;
```

Reply:
Yes this is better, I will change it.